webapps-backup/src/main/java/io/camunda/webapps/backup/BackupServiceImpl.java

```diff
@@ -93,18 +96,19 @@ public TakeBackupResponseDto takeBackup(final TakeBackupRequestDto request) {
 
   @Override
   public GetBackupStateResponseDto getBackupState(final Long backupId) {
-    return repository.getBackupState(getRepositoryName(), backupId);
+    return repository.getBackupState(backupProps.repositoryName(), backupId);
   }
 
   @Override
   public List<GetBackupStateResponseDto> getBackups() {
-    return repository.getBackups(getRepositoryName());
+    return repository.getBackups(backupProps.repositoryName());
   }
 
   TakeBackupResponseDto scheduleSnapshots(final TakeBackupRequestDto request) {
-    final String repositoryName = getRepositoryName();
+    final String repositoryName = backupProps.repositoryName();
     final int count = getIndexPatternsOrdered().length;
     final List<String> snapshotNames = new ArrayList<>();
+    // FIXME potential issues
```

Comment:
:question: What's this here for?

Reply:
I'll remove it, I put it when we were wondering what to do about different versions in the backup name between tasklist & operate